### DISCUSSION TITLE
Please make it so I can turn off the adding a desktop icon

### DISCUSSION THREAD

**Discussion by cat-phish** [f1multiviewer/issue-tracker#403] [Feature requests]
Every time the app updates, basically every race (I appreciate the active development) but it adds the icon back to my desktop. It's gotten to the point that I've just disabled write permissions to my desktop to try to fix it because it's such a recurring problem.

**Comment by LapsTimeOFF** [comment]
Hey this is beyond our control, if this is still happening please raise an issue in the Issues tab.
